command/agent/config/test-fixtures/config-template_config-empty.hcl

```diff
@@ -0,0 +1,13 @@
+vault {
+  address = "http://127.0.0.1:1111"
+  retry {
+    num_retries = 5
+  }
+}
+
+template_config {}
```

Comment:
Is an empty `template_config` now required or can this be omitted if templating is not being configured?

Reply:
It can be omitted entirely.